tests/by-util/test_tail.rs

```diff
@@ -169,37 +172,9 @@ fn test_nc_0_wo_follow() {
         .succeeded();
 }
 
+// TODO: Add similar test for windows
 #[test]
-#[cfg(all(unix, not(target_os = "freebsd")))]
-fn test_nc_0_wo_follow2() {
```

Comment:
Why is this removed?

Reply:
Because the test is platform specific in contrast to `test_nc_0_wo_follow` and it doesn't add value. In my eyes the `test_nc_0_wo_follow` test was sufficient. However, it could be more precise with the debug_assert(). If you like to, I can readd it.

Reply:
I can't follow your reasoning.
> Because the test is platform specific in contrast to test_nc_0_wo_follow

This should not be a reason to remove a test.

> and it doesn't more information.

But the two tests check different things.